internal/shared/exec/exec.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 const (
-	defaultTimeout = 10 * time.Second
+	defaultTimeout = 90 * time.Second
```

Comment:
Helm dependencied fetch takes time to had to increase it, also looked in argo-cd that is also using 90s [here](https://github.com/argoproj/argo-cd/blob/3fee8cbf815fb540aec4b0e0a248d28cc84a86a6/util/exec/exec.go#L36)